lib/assessment.rb

```diff
@@ -5,7 +5,7 @@ class Assessment
   # enumerable supplied.  Assume each entry implements a length
   # method that returns a Fixnum.
   def longest(enum)
-    # Your code goes here
+    enum.max_by { |x| x.length }.to_i
```

Comment:
I figured the to_i which as I understand it is to integer would return a
number since we wanted the number of the longest string not the string
itself.  Could have misunderstood.
On Oct 23, 2015 7:22 PM, "Matt Brendzel" <notifications@github.com> wrote:

> In lib/assessment.rb
> <https://github.com/ga-wdi-boston/ruby-enumerable-assessment/pull/10#discussion_r42924418>
> :
>
> > @@ -5,7 +5,7 @@ class Assessment
> >    # enumerable supplied.  Assume each entry implements a length
> >    # method that returns a Fixnum.
> >    def longest(enum)
> > -    # Your code goes here
> > +    enum.max_by { |x| x.length }.to_i
>
> Not the expected answer, but I think this will work. Why the to_i, though?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ga-wdi-boston/ruby-enumerable-assessment/pull/10/files#r42924418>
> .
>